view/frontend/web/template/payment/google-pay-form.html

```diff
@@ -38,7 +38,7 @@
         </label>
     </div>
 
-    <div class="payment-method-content">
+    <div class="payment-method-content" data-bind="event: { change: onPaymentMethodContentClick }">
```

Comment:
what if the google pay is the only enabled payment method, is the button by default visible and enabled or is it disabled already?

Reply:
It is disabled by default. Which might cause problems when there are no agreements in place, now that I think about it.